.gitignore

```diff
@@ -102,3 +102,7 @@ dist
 
 # TernJS port file
 .tern-port
+
+yarn.lock
+
+package-lock.json
```

Comment:
Hey everyone

I understand the project uses npm and am some  use use yarn

Due to differences in package manager I ask we keep the lock files ignored

Due to the nature of the project not keeping the lock is not fatal

Reply:
Bro @gsonly , what are you think about this, am okay with this

Reply:
I also prefer yarn... its fine by me.